spec/features/admin/index_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe 'index.html.erb' do
+  describe 'visit' do
+    it 'displays a dashboard' do
+      visit '/admin'
+
+      within('h1#dash') do
```

Comment:
👍  nice use of `within` here for testing - this is a great idea to get started with now since these pages are going to get crowded! I'll be sure to incorporate that moving forward in my work also.